### PR TITLE
Add custom bundle detection to debug command

### PR DESCRIPTION
**Pull request by OmegaSunkey**
This PR adds custom bundle boolean to the /debug command for well, support reasons.

### CODE REVIEW

File: src/core-plugins/CoreCommands.ts [hunk 1]
```diff
@@ -83,7 +86,7 @@ export default class CoreCommands extends Plugin {
                 MessageActions.sendMessage(ctx.channel.id, {
                     content: `**Debug Info:**
                         > Discord: ${DebugInfo.discordVersion}
-                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins)
+                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins) Custom Bundle: ${customBundle}
```
Comment (TymanWasTaken) [inline src/core-plugins/CoreCommands.ts:89]: I feel like this should be on a new line just to keep everything consistent

File: src/core-plugins/CoreCommands.ts [hunk 2]
```diff
@@ -5,6 +5,9 @@ import { Plugin } from "../entities/Plugin";
 import { getByProps, Locale, MessageActions } from "../metro";
 import { DebugInfo } from "../utils/debug/DebugInfo";
 import { makeAsyncEval } from "../utils/misc";
+import { exists, externalStorageDirectory } from "../native/fs";
+
+const customBundle: boolean = AliuFS.exists(externalStorageDirectory + "/AliucordRN/Aliucord.js.bundle");
```
Comment (TymanWasTaken) [inline src/core-plugins/CoreCommands.ts:10]: You shouldn't need to put explicit types here

**Review by TymanWasTaken — CHANGES_REQUESTED** [review]
Also what gh actions said

File: src/core-plugins/CoreCommands.ts (hunk 2 above)
Comment (OmegaSunkey) [reply src/core-plugins/CoreCommands.ts]: thought ts was all about explicit static types

File: src/core-plugins/CoreCommands.ts (hunk 1 above)
Comment (OmegaSunkey) [reply src/core-plugins/CoreCommands.ts]: done

File: src/core-plugins/CoreCommands.ts [hunk 3]
```diff
@@ -83,7 +86,8 @@ export default class CoreCommands extends Plugin {
                 MessageActions.sendMessage(ctx.channel.id, {
                     content: `**Debug Info:**
                         > Discord: ${DebugInfo.discordVersion}
-                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins)
+                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins
```
Comment (rushiiMachine) [inline src/core-plugins/CoreCommands.ts:89]:
```suggestion
                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins)
```

Comment (OmegaSunkey) [reply src/core-plugins/CoreCommands.ts:89]: LMAO I HATE GITHUB.DEV how did i remove that

**Review by TymanWasTaken — APPROVED** [review]

**Review by rushiiMachine — APPROVED** [review]

**Pull request merged**